internal/cmd/cmd_test.go

```diff
@@ -1008,7 +1008,7 @@ func testDownload(t *testing.T) {
 	// download checks if protoc is already downloaded to the cache location
```

Comment:
Should this test name change?  i.e. `testCacheUpdate`

Reply:
This isn't actually a test, this just populates the cache, it's a helper function